test/demo/connection.inttest.ts

```diff
@@ -210,9 +210,9 @@ async function navigateTo(target: Identity, url: string): Promise<void> {
     await delay(Duration.PAGE_NAVIGATE);
 }
 
-async function getIntentListeners(intentType: string): Promise<Application[]> {
-    return ofBrowser.executeOnWindow(SERVICE_IDENTITY, function (this: ProviderWindow, type: string): Promise<Application[]> {
```

Comment:
this change makes `Application` an unused import

Reply:
✅